.travis.yml

```diff
@@ -7,8 +7,22 @@ compiler:
 addons:
   apt:
     packages:
+        - libjson-c-dev
+        - libkeyutils-dev
+        - libkmod-dev
+        - libnuma-dev
+        - libudev-dev
         - tcl8.5
 
+before script:
+  - export CC=`which $CC`
+  - wget https://github.com/pmem/ndctl/archive/v68.tar.gz
+  - tar -xzvf v68.tar.gz
+  - pushd ndctl-68 && ./autogen.sh && ./configure --disable-docs --prefix=/usr && make && sudo make install && popd
+  - wget https://github.com/memkind/memkind/archive/master.tar.gz
```

Comment:
use a release version may better

Reply:
@hongliuliao 
I used the memkind v1.10.1 - release candidate version, because this version contains fix not available in last official version 1.10.0.
After next official memkind release we update here link to official 1.10.1 version.

Reply:
It's ok, we can use master archive temporary before 1.10.1 version release.